src/components/scenes/SwapSettingsScene.tsx

```diff
@@ -41,6 +42,7 @@ type Props = StateProps & DispatchProps & ThemeProps
 
 interface State {
   enabled: { [pluginId: string]: boolean }
+  swapConfigUnsubscribeFns: CallbackRemover[]
```

Comment:
On a class-style component, you can add things as properties of `this`, instead of putting them into `State`. I believe `swapConfigUnsubscribeFns` should be a `this` property, not a state property, since it doesn't affect rendering.

In the hooks world we wouldn't put this into a `useState` - it would be managed internally by the `useEffect` hook's own storage.